tesseract_motion_planners/ompl/src/ompl_motion_planner.cpp

```diff
@@ -293,14 +192,9 @@ PlannerResponse OMPLMotionPlanner::solve(const PlannerRequest& request) const
     cached_simple_setups.reserve(move_instructions.size());
 
   // Transform plan instructions into ompl problem
-  unsigned num_output_states = 1;
-  long start_index{ 0 };
-  std::size_t segment{ 1 };
```

Comment:
Note: segment is the same as the iterator variable `i`, so no need to make something separate that needs to be incremented separately

Reply:
It is not the same as 'i'. The 'i' increments for every move instruction in a segment. A segment is incremented when a constrained waypoint is reached but can have any number of unconstrained waypoints in-between.